app/views/admin/edition_translations/edit.html.erb

```diff
@@ -18,6 +18,8 @@
           heading_size: "l",
           value: @translated_edition.title,
           error_items: errors_for(form.object.errors, :title),
+          right_to_left: form.object.rtl_locale?(translation_locale),
```

Comment:
I think we can use this method https://github.com/alphagov/whitehall/blob/main/app/models/edition/translatable.rb#L51-L53

So  it would just be `form.object.rtl?`

Reply:
Actually, it seems that method returns a value based on the `primary_local`, which is always `en`. I can add a new method like below which does what is required. Does that seem like a better option? 

```
def translation_rtl?
  Locale.find_by_code(translation_locale.code).rtl?
end
```

Reply:
Oh you're absolutely correct. I think you were bang on before then, but i think it should probably live in the translatable concern next to the `rtl?` method. Wdyt?

Reply:
Or alternatively
```
def translation_rtl?
  Locale.new(translation_locale).rtl?
end
```

Reply:
@davidgisbey I've updated that to add the new method in translatable and use that in the view. I'll squash the commits if that looks OK.